src/com/adgoji/mollie/client.clj

```diff
@@ -84,12 +84,12 @@
          :always         (response-transformer)
          check-response? (spec/check spec))
        (let [anomaly (condp some #{status}
-                       #{400 401} ::anomalies/incorrect
-                       #{403}     ::anomalies/forbidden
-                       #{404 410} ::anomalies/not-found
-                       #{405}     ::anomalies/unsupported
-                       #{500}     ::anomalies/fault
-                       #{503}     ::anomalies/busy)
+                       #{400 401 422} ::anomalies/incorrect
+                       #{403}         ::anomalies/forbidden
+                       #{404 410}     ::anomalies/not-found
+                       #{405}         ::anomalies/unsupported
+                       #{500}         ::anomalies/fault
+                       #{503}         ::anomalies/busy)
```

Comment:
Btw, you can consider using `case` here:

```clojure
(case status
  (400 401 422) ::anomalies/incorrect
  403           ::anomalies/forbidden
  (404 410)     ::anomalies/not-found
  405           ::anomalies/unsupported
  500           ::anomalies/fault
  503           ::anomalies/busy)
```

Reply:
Wow, I didn't know `case` works on lists.

Reply:
<img width="452" alt="image" src="https://github.com/AdGoji/mollie/assets/100711682/c6d782ab-04af-4843-a9de-9019c5b1481d">

Indeed! I should read the docs more carefully :) Thanks!